src/wrapper-types.jl

```diff
@@ -0,0 +1,144 @@
+export @symbolic_wrap, @wrapped
+
+using ExprTools
```

Comment:
This conflicts with `splitdef` imported from MacroTools.jl

Reply:
I will remove ExprTools dependency